Description: Don't run the test suite in random order.
 Can be dropped in 3.13.
Bug-Upstream: https://github.com/python/cpython/issues/110164

--- a/Tools/scripts/run_tests.py
+++ b/Tools/scripts/run_tests.py
@@ -56,7 +56,6 @@ def main(regrtest_args):
     args.extend(test.support.args_from_interpreter_flags())
 
     args.extend(['-m', 'test',    # Run the test suite
-                 '-r',            # Randomize test order
                  '-w',            # Re-run failed tests in verbose mode
                  ])
     if sys.platform == 'win32':
